services/core-web/src/components/Forms/ExplosivesPermit/ExplosivesPermitFormNew.tsx

```diff
@@ -119,11 +121,117 @@ export const ExplosivesPermitFormNew: FC<ExplosivesPermitFormNewProps> = (props)
     "now_application_guid"
   );
 
-  const isHistoric = !props.initialValues?.explosives_permit_id && props.isPermitTab;
+  const [isHistoric, setIsHistoric] = useState<boolean>(!initialValues?.explosives_permit_id && props.isPermitTab);
   const isESUP = props.userRoles.includes(USER_ROLES[Permission.EDIT_EXPLOSIVES_PERMITS]);
 
   const disabled = props.isProcessed; // props.isProcessed && !hasEditPermission;
+
+  const [radioSelection, setRadioSelection] = useState<number>(props.isPermitTab ? 1 : 2);
+  const [parentView, setParentView] = useState<boolean>(true);
+  const [isAmend, setIsAmend] = useState<boolean>(false);
+  const { isFeatureEnabled } = useFeatureFlag();
+
+  const handleRadioChange = (e) => {
+    setRadioSelection(e.target.value);
+    setIsHistoric(e.target.value == 1);
+    setIsAmend(e.target.value==3);
```

Comment:
yeeeeah please lint these files. For the PRs of the future.

Reply:
Updated. Please verify.